README.md

```diff
@@ -60,31 +63,31 @@ any work.
 We see project ideas coming from 2 main sources:
 
 #### Open Applications
-* If you have an idea for the Radicle or web3/FOSS ecosystem, we'd love for you to apply. 
+* If you have an idea for the Radicle, Drips or web3/FOSS ecosystem, we'd love for you to apply. 
 * You can file an application using the [template](grants/applications/template.md). More on this in the 
 [process](#process) section.
 
 #### [Requests For Proposals](rfps) (RFPs) 
-* The Core Development team may create an RFP, but anyone from the community may as well. 
+* The Core Development team may create an RFP, but anyone from the community may as well. This helps signal to grant applicants what is most needed in the ecosystem.
 * Anyone is welcome to apply to complete the work associated with an RFP.
 * We accept applications to tackle RFPs from multiple individuals and teams. This means you may be in competition for a 
 grant. Where it makes sense, we may also award one grant across multiple teams.
 
 > 💰 If you write an RFP, you may collect a referral bonus equal to 5% of the total size of any 
-associated grant that is accepted _**and**_ completed.
+associated grant in $RAD that is accepted _**and**_ completed.
 
-No matter the source of the project idea, the entire application process will be transparent and public on Radicle 
-channels.
+No matter the source of the project idea, the entire application process will be transparent and public on Radworks 
```

Comment:
This "Radworks" should actually be "Radicle" as it's talking about the RFPs and grantee's code being hosted on Radicle

Maybe we should reword like this:
```
Regardless of the source of the project idea, the entire RFP and application process will be public on Radicle.
```